rocket-mission-persistence/src/main/java/cz/muni/fi/entity/Mission.java

```diff
@@ -1,21 +1,159 @@
 package cz.muni.fi.entity;
 
+import com.sun.istack.NotNull;
+import cz.muni.fi.enums.MissionProgress;
+
+import javax.persistence.Column;
 import javax.persistence.Entity;
 import javax.persistence.GeneratedValue;
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
+import javax.persistence.ManyToMany;
+import javax.persistence.OneToMany;
+import java.time.ZonedDateTime;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
+
+/**
+ * Class representing mission entity.
+ *
+ * @author Martin Kazimir
+ */
 
 @Entity
 public class Mission{
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
 
+    @NotNull
+    @Column(unique = true, nullable = false)
+    private String name;
+
+    private String destination;
+    private MissionProgress missionProgress;
+
+    @ManyToMany
+    private Set<User> users = new HashSet<>();
+
+    @OneToMany
+    private Set<Rocket> rockets = new HashSet<>();
+
+    @ManyToMany
```

Comment:
Tady jsme se teda domluvili, že to bude @OneToMany či?

Reply:
jop, len som to uz vecer nezmenil